node/src/main.rs

```diff
@@ -399,18 +399,20 @@ fn async_main() -> impl Future<Item = (), Error = ()> + Send + 'static {
     let mut subscription_server =
         GraphQLSubscriptionServer::new(&logger, graphql_runner.clone(), store.clone());
 
-    // Create Ethereum block ingestor
-    let block_ingestor = graph_datasource_ethereum::BlockIngestor::new(
-        store.clone(),
-        transport.clone(),
-        50, // ancestor count, which we could make configuable
-        logger.clone(),
-        block_polling_interval,
-    )
-    .expect("failed to create Ethereum block ingestor");
+    if env::var_os("DISABLE_BLOCK_INGESTOR").unwrap_or("".into()) != "true" {
```

Comment:
We can just test if the var is present at all, what do you think?

Reply:
Ah, I did it this way in case someone tries to do `DISABLE_BLOCK_INGESTOR=false` actually

Reply:
Makes sense, though there's also the case of trying `DISABLE_BLOCK_INGESTOR=1` or `DISABLE_BLOCK_INGESTOR=yes` or something. But either way works.

Reply:
Yeah, that's true... :smile: